tests/nightly/test_large_vector.py

```diff
@@ -710,6 +710,39 @@ def test_full():
     assert a[-1] == 3
 
 
+def test_regression():
+    shape = (LARGE_X, )
+
+    def check_regression(symbol, forward, shape):
+        # init executor
+        data_s = mx.symbol.Variable('data')
+        label_s = mx.symbol.Variable('label')
+        out_s = symbol(data=data_s, label=label_s)
+        exe = out_s.simple_bind(ctx=mx.cpu(0), data=shape, label=shape)
```

Comment:
what about infer shape. is it tested with bigger sizes ?

Reply:
haven't checked infer shape specifically. but we are checking output of regressions with numpy outputs for entire vector (so i am guessing shape check indirectly happens)
i can add an explicit check for shape if it is really needed.

Reply:
no i mean calling sym.infer_shape with large sizes. do we have a test for that ? if not we should add it. many users use this api

Reply:
oh we don't have test for it. adding now.

Reply:
Added the test.

Reply:
cool, does it work ?

Reply:
```
nosetests tests/nightly/test_large_vector.py:test_infer_shape
.
----------------------------------------------------------------------
Ran 1 test in 0.001s

OK
```